src/paste.ts

```diff
@@ -3,15 +3,25 @@ import {
   setSelectionRangeIfNecessary,
   calculateNewValue,
   eventWrapper,
+  isDisabled,
 } from './utils'
 
+interface pasteOptions {
+  initialSelectionStart?: number
+  initialSelectionEnd?: number
+}
+
 function paste(
-  element,
-  text,
-  init,
-  {initialSelectionStart, initialSelectionEnd} = {},
+  element: HTMLInputElement | HTMLTextAreaElement,
```

Comment:
This type definition makes it impossible to use elements returned by RTS queries, without type casting.

see: https://github.com/testing-library/user-event/issues/648

Reply:
Here's a PR to fix this: https://github.com/testing-library/user-event/pull/649